ranking-service/src/main/resources/properties/jpa.yml

```
@@ -0,0 +1,12 @@
+spring:
+  config.activate.on-profile: local
+  jpa:
+    generate-ddl: true
+    show-sql: true
+    properties:
+      hibernate:
+        format_sql: true
+    hibernate:
+      ddl-auto: create
+    defer-datasource-initialization: true
+    open-in-view: false
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)